File: split-0s-1s/ruby/spec/main_spec.rb
require 'main.rb'
require 'byebug'

describe "Segregate 0's and 1's" do
  it "does example case" do
    expect(
      segregate([0, 1, 0, 1, 0, 0, 1, 1, 1, 0])
    ).to eq [0, 0, 0, 0, 0, 1, 1, 1, 1, 1]
  end
end

File: split-0s-1s/js/__tests__/main.spec.js
const segregate = require('../lib/main')

describe('sort 0s and 1s', () => {
  it('does sample cases', () => {
    expect(
      segregate([0, 1, 0, 1, 0, 0, 1, 1, 1, 0])
    ).toEqual([0, 0, 0, 0, 0, 1, 1, 1, 1, 1])
  })
})
